docs/windows-broker.md

```diff
@@ -181,8 +205,10 @@ See https://aka.ms/gcm/wamadmin for more information.
 
 In order to fix the problem, there are a few options:
 
-1. Run Git or Git Credential Manager from non-elevated processes.
-2. Disable the broker by setting the
+1. Update to the [latest Git for Windows][git-for-windows-latest]
+   **(recommended)**.
+2. Run Git or Git Credential Manager from non-elevated processes.
```

Comment:
```suggestion
2. Run Git or Git Credential Manager from a non-elevated processes.
```

Reply:
I'm using the plural "processes" here, so "a" isn't correct grammar.